frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/provision-source.tsx

```diff
@@ -77,9 +94,18 @@ export const ProvisionSourceComponent: React.FC<ProvisionSourceComponentProps> =
               placeholder={getPlaceholder(VMSettingsField.PROVISION_SOURCE_TYPE)}
               isDisabled={!!provisionSourceValue}
             />
-            {(sources || []).map((source) => (
-              <FormSelectOption key={source} value={source} label={source} />
-            ))}
+            {(sources || []).map((source) => {
+              if (source === ProvisionSource.PXE.toString() && !hasNADs) {
+                return (
+                  <FormSelectOption
+                    key={source}
+                    value={SELECT_PXE_NAD_ERROR}
```

Comment:
the value should be an enum and is not for storing errors. This can get mishandled pretty easily in the redux store. Especially with changing requirements.

This will also not work well with the template flow.

Reply:
not changing the value now